src/commands/protocols/common/network-protocol-command.js

```diff
@@ -43,6 +47,16 @@ class NetworkProtocolCommand extends Command {
         return [];
     }
 
+    async getBatchSize() {
+        // overridden by subclasses
+        return 0;
```

Comment:
let's throw error here, since this method is required to be overridden in subclasses, if it throws an error it will be easier for us to catch what is the issue

Reply:
good approach, I'll change this